content/blog/Installation.md

```diff
@@ -1,6 +1,113 @@
 ---
-date: 2024-02-20
-title: Installing orchestrator using helm chart on OpenShift
+date: 2024-03-13
+title: Installing the Orchestrator on existing RHDH instance
 ---
 
-{{< remoteMD "https://github.com/parodos-dev/orchestrator-helm-chart/blob/main/README.md?raw=true" >}}
+When RHDH is already installed and used, it is required to avoid re-installing it by the Helm chart.
+There are few steps required add the Orchestrator to such environment:
+1. Install the backing services by the Helm chart by disabling the RHDH installation
+2. Manually patching the existing RHDH ConfigMap resources with the Orchestrator plugin configuration.
+3. Import the Orchestrator software templates into Backstage catalog
+
+To install the backing services without RHDH use the option `--set rhdhOperator.enabled=false`
+A complete command will look like:
+`helm upgrade -i --debug orchestrator orchestrator --set rhdhOperator.enabled=false` 
+Resulting in the installation of the Sonataflow Operator, Serverless Operators.
+Alternately, those operators can be installed directly from the operator catalog.
+
+In RHDH installation, there are two major ConfigMaps to edit, under the `backstage-system` or the `rhdh-operator` namespaces:
+`dynamic-plugins` - contains the configuration to enable and set the dynamic plugins.
+To add the orchestrator plugins, add into the `dynamic-plugins` configmap the following:
```

Comment:
To better understand: is it ok to append the following code? Or is there a specific place in the CM ?

Reply:
there is one section in that configmap where dynamic plugins configuration are added. order doesn't matter.